tests/components/reolink/test_media_source.py

```diff
@@ -130,6 +130,7 @@ async def test_browsing(
     """Test browsing the Reolink three."""
     entry_id = config_entry.entry_id
     reolink_connect.api_version.return_value = 1
+    reolink_connect.model = "Reolink TrackMix PoE"
```

Comment:
Non blocking: Would be good to add a test that verifies the lens is included in the name too.

Reply:
I added a assertion to check the device name, thanks for the suggestion.